content/en/users/cloud-compute/faq/_index.md

```diff
@@ -225,3 +225,28 @@ keys in the VMs to be able to access.
 If you don\'t have public IPs for all the VMs to be managed, you can also use
 one as a gateway as described in the
 [Ansible FAQ](https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#how-do-i-configure-a-jump-host-to-access-servers-that-i-have-no-direct-access-to).
+
+### How can I release resources without destroying my data?
+
+Whenever you delete a VM, the ephemeral disks associated with it will be also
+deleted. If you don't plan to use your VM for some time, there are several
+ways to release resources consumed by the VM (e.g. CPU, RAM) and  recover the
+the data or boot your VM in a previous state when you need it back. We list
+below the main strategies you can use:
+
+- Use a volume to store the data to be kept: Check the [Storage section of the
+  documentation](../storage) to learn how to use volumes. If you start your VM
+  from a volume, the VM can be destroyed and recreated easily. OpenStack
+  documentation cover how to [start a VM from a volume with CLI](https://docs.openstack.org/nova/latest/user/launch-instance-from-volume.html)
+  or [using the Horizon dashboard](https://docs.openstack.org/horizon/latest/user/launch-instances.html)
+
+- Suspend or shelve instances: Suspending a VM will pause a VM, releasing CPU
+  and memory, and allowing to resume later in time at the exact same state.
+  Shelving shuts down the VM, thus RAM contents will be lost but disk will be
+  kept. This releases more resources from the provider while still allows to
+  easily boot the VM back without losing disk contents.
+
+- Create snapshot of instances: a snapshot will create a new VM image at your
```

Comment:
```suggestion
- Create snapshot of instance: a snapshot will create a new VM image at your
```

Reply:
I understand that either it should have been snapshots of instances or have instance singular, thus suggested to just use instance.